Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Settings/History/Object/ObjectVersionViewModel.swift

```diff
@@ -17,18 +18,20 @@ final class ObjectVersionViewModel: ObservableObject {
     @Published var screenData: EditorScreenData?
     let data: ObjectVersionData
     
-    private let openDocumentProvider: OpenedDocumentsProviderProtocol = Container.shared.documentService()
-    
     init(data: ObjectVersionData) {
         self.data = data
-        self.screenData = currentScreenData()
     }
     
-    private func currentScreenData() -> EditorScreenData? {
+    func setupObject() async {
+        self.screenData = await currentScreenData()
+    }
+    
+    private func currentScreenData() async -> EditorScreenData? {
```

Comment:
why async?

Reply:
Not needed, forgot to remove, fixed in the following PR